src/main/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImpl.java

```diff
@@ -338,7 +339,18 @@ protected void cleanPayeeACHAccountExtractDetail(PayeeACHAccountExtractDetail de
             if (!StringUtils.isNumeric(detail.getBankAccountNumber())) {
                 LOG.error(logMessageStarter + " is not numeric after cleaning");
             }
+
+            detail.setBankName(extractBankName(detail));

```

Comment:
This statement is currently inside the "if" block that checks for a non-numeric bank account number. Should this line be moved outside of that "if" block so that the cleaning of the bank name does not depend on the account number formatting?